rules/rules.libsonnet

```diff
@@ -58,6 +58,37 @@
           },
         ],
       },
+      {
+        name: 'kube-scheduler.rules',
+        rules: [
+          {
+            record: 'cluster:%s:quantile' % metric,
+            expr: |||
+              histogram_quantile(%(quantile)s, sum(%(metric)s_microseconds_bucket{%(kubeSchedulerSelector)s}) BY (le, cluster)) / 1e+06
+            ||| % { quantile: quantile, metric: metric, kubeSchedulerSelector: $._config.kubeSchedulerSelector },
+            labels: {
+              quantile: quantile,
+            },
+          }
+          for quantile in ['0.99', '0.9', '0.5']
+          for metric in ['scheduler_e2e_scheduling_latency', 'scheduler_scheduling_algorithm_latency', 'scheduler_binding_latency']
+        ],
+      },
+      {
+        name: 'kube-apiserver.rules',
+        rules: [
+          {
+            record: 'apiserver_latency_seconds:quantile',
+            expr: |||
+              histogram_quantile(%(quantile)s, rate(apiserver_request_latencies_bucket{%(kubeApiserverSelector)s}[5m])) / 1e+06
```

Comment:
And this a sum?

Reply:
Also https://github.com/kubernetes-incubator/apiserver-builder/blob/master/vendor/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L56 implies this is in microseconds, not seconds.